### PR TITLE
Add chains and lanterns to the iron forge

### PR DESCRIPTION
**Pull request by wingzero54**

### CODE REVIEW

File: templates/public/plugins/FactoryMod/config.yml.j2 [hunk 1]
```diff
@@ -3375,6 +3376,21 @@ recipes:
...
         amount: 256
+  make_chains:
+    production_time: 4s
+    name: Make Chains
+    type: PRODUCTION
+    input:
+      iron:
+        material: IRON_INGOT
+        amount: 16
+      iron:
```
Comment (Maxopoly) [inline templates/public/plugins/FactoryMod/config.yml.j2:3387]: I'd rather have a bit of additional iron ingot cost than including nuggets, crafting those is just annoying

Comment (Maxopoly) [reply templates/public/plugins/FactoryMod/config.yml.j2:3387]: Also both sections under input have the same identifier, which probably isn't gonna work

Comment (wingzero54) [reply templates/public/plugins/FactoryMod/config.yml.j2:3387]: Roger that. I also realized Lanterns would be a good addition as they're primarily made of iron, I'm going to include those as well.

**Comment by Maxopoly** [comment]
Nice

**Pull request merged**